### PR TITLE
(feature) [241] filtering by location

### PR DESCRIPTION
**Pull request by despo**
- switches to Google's geolocation API
- configures API token

### CODE REVIEW

**Comment by despo** [comment]
[Filtering by location](https://trello.com/c/m1qXKSOq/241-filtering-by-location)

**Comment by despo** [comment]
This has been deployed to Edge https://tvs.edge.dxw.net/

As we only have one visible result the best you can do is filter by  Hartlepool and use a radius of 5miles as Hartlepool's coordinates are about 1.5 miles away from the displayed job listing.

File: config/locales/en.yml [hunk 1]
```diff
@@ -150,8 +150,8 @@ en:
       application_link: 'This is the url linking the job to your existing application page'
     filters:
       title: 'Search job listings'
-      location: 'Postcode'
-      location_hint: 'Full UK postcode'
+      location: 'Location'
+      location_hint: 'Town, postscode or region'
```
Comment (tahb) [inline config/locales/en.yml]: Typo here in an extra `s`, it should be `postcode`.

**Comment by despo** [comment]
@tahb updated!

**Pull request merged**